### ISSUE TITLE
Local directory nuget sources return "Cannot determine latest version!" while trying to determine latest version.

### ISSUE THREAD

**Issue by sideproject** [jerriep/dotnet-outdated#109]
Version: 2.1.0-rtm.169+9c231c845053581c7e2fbe80d4531a029105f597

While running `dotnet outdated` on a solution that uses a local directory nuget source I see the text __Cannot determine latest version!__ and there is an error at the bottom saying __Errors occurred while analyzing dependencies for some of your projects. Are you sure you can connect to all your configured NuGet servers?__

Here is a sample nuget.config that will add a local directory nuget source:
```<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="LocalDirectory1" value="C:\nugetpackages" />
  </packageSources>
</configuration>
```

I was able to add local directory support with the following patch:
```
diff --git a/src/DotNetOutdated/Services/NuGetPackageInfoService.cs b/src/DotNetOutdated/Services/NuGetPackageInfoService.cs
index 0d079e5..9136936 100644
--- a/src/DotNetOutdated/Services/NuGetPackageInfoService.cs
+++ b/src/DotNetOutdated/Services/NuGetPackageInfoService.cs
@@ -103,6 +103,10 @@ namespace DotNetOutdated.Services
                             {
                                 allVersions.Add(packageSearchMetadataV2Feed.Version);
                             }
+                            else if (m is LocalPackageSearchMetadata localPackageSearchMetadata)
+                            {
+                                allVersions.Add(localPackageSearchMetadata.Identity.Version);
+                            }
                         };
                     }
                 }
```

I'm happy to create a pull request if this is something you would consider accepting.

**Comment by jerriep** [comment]
@sideproject I'll be happy to accept a PR for this, thanks!

**Comment by jerriep** [comment]
Closed with #110
